### ISSUE TITLE
Which package to use from npm: monaco-languageclient or @codingame/monaco-languageclient?

### ISSUE THREAD

**Issue by pplonski** [TypeFox/monaco-languageclient#304]
This repository name is `monaco-languageclient` but in the example, the `@codingame/monaco-languageclient` is used? Why? I can't find the repository for `@codingame/monaco-languageclient` - what is the license for it? Which one should be used?

I'm building an example of how to use monaco-editor in Electron with Python LSP.  Here is my repo https://github.com/pplonski/electron-monaco-python-lsp I used `@codingame` packages because otherwise, I got a lot of errors. The example is a simple proof-of-concept for a larger project (open-source IDE for data science). May I ask what happened with R-BRAIN project? The website (r-brain.io) is not working.

Anyway, Thank you for this repo because I was able to run full completion for Python in Electron :) Thanks!

**Comment by CGNonofr** [comment]
Hello,

This project is no longer maintained by it's original maintainer.

I'm now the main maintainer but I only got the access on the github, not the npm repo, so the package is now published in the repo of my organization. The github will probably move in the future as well.

TL;DR: `@codingame/monaco-languageclient` is the one to use

**Comment by pplonski** [comment]
Thank you! @CGNonofr you are doing a great job!

It will be good to move the repository. It will be less confusing.

I've seen also your packages:
- https://github.com/CodinGame/monaco-editor-wrapper
- https://github.com/CodinGame/monaco-editor-react
- https://github.com/CodinGame/monaco-languageclient-wrapper
- https://github.com/CodinGame/monaco-languageclient-react

Are they ready to use? Do you need help with those packages?

**Comment by CGNonofr** [comment]
Those are some internal stuff that was open-sourced recently. We're not really expecting people to use it but feel free to do so and help us if you feel like it!
